Makefile

```diff
@@ -229,7 +227,7 @@ protobuf:
 .PHONY: .go-version
 .go-version:
 	@actual=$$($(GO) version); \
-	echo "$${actual}" | grep -q $(GOVERS) || \
+	echo "$${actual}" | grep -q '\b$(GOVERS)\b' || \
```

Comment:
It's already possible for humans to override this. I'm open to concrete suggestions, but I think this will be such a rare issue in practice that it's not worth optimizing for.